packages/server/src/requestPipeline.ts

```diff
@@ -412,21 +417,16 @@ export async function processGraphQLRequest<TContext extends BaseContext>(
       );
       // XXX: This cast is pretty sketchy, as other error types can be thrown
       // in the try block!
-      return await sendErrorResponse(executionError as GraphQLError);
-    }
-  }
-
-  if (internals.formatResponse) {
-    const formattedResponse: GraphQLResponse | null = internals.formatResponse(
-      response,
-      requestContext,
-    );
-    if (formattedResponse != null) {
-      response = formattedResponse;
+      return await sendErrorResponse(
+        executionError as GraphQLError,
+        undefined,
+        newHTTPGraphQLHead(500),
+      );
     }
   }
 
-  return sendResponse(response);
+  await invokeWillSendResponse();
+  return;
 
   function parse(query: string, parseOptions?: ParseOptions): DocumentNode {
     return graphqlParse(query, parseOptions);
```

Comment:
Not related to your changes but why does this exist?

Reply:
Before Jesse removed the old extension stack API this function called it. (https://github.com/apollographql/apollo-server/commit/05f0d2f799dfc81cda7435093b3d6b67000c412c) Sure, let's inline, why not.